services/apps/portal/src/application/application-document/application-document.controller.ts

```diff
@@ -78,6 +80,17 @@ export class ApplicationDocumentController {
     return await this.applicationDocumentService.getInlineUrl(document);
   }
 
+  @Patch('/application/:fileNumber')
+  async update(
+    @Param('fileNumber') fileNumber: string,
+    @Req() req,
+    @Body() body: ApplicationDocumentUpdateDto[],
+  ) {
+    await this.applicationService.verifyAccess(fileNumber, req.user.entity);
+    await this.applicationDocumentService.update(body, fileNumber);
+    return;
```

Comment:
return nothing?

Reply:
What should I return?

Reply:
you could do something like return this.applicationDocumentService.update(body, fileNumber). Where update on service side does return repo.update(array)

Reply:
Done